bottlecap/src/metrics/aggregator.rs

```diff
@@ -1,15 +1,19 @@
 //! The aggregation of metrics.
 
-use crate::metrics::metric;
-use crate::metrics::{
-    constants, datadog, errors,
-    metric::{Metric, Type},
+use crate::{
+    metrics::{constants, datadog, errors, metric},
+    tags::provider,
 };
+use datadog::Metric as DatadogMetric;
+use metric::{Metric, Type};
```

Comment:
Double import for `metric`, `datadog`, and `metrics`

Reply:
Not sure what is a double import, but we are using 2 different metrics structs. I changed names and nested everything for clarity